toolkit/pythonit_toolkit/api/client.py

```diff
@@ -0,0 +1,54 @@
+import os
```

Comment:
I am not sure on the namespace of the client, `api/` I would prefer `service_client` maybe? it needs to have a name that explain what it does